boot.py

```diff
@@ -1,20 +1,24 @@
-from storage import disable_usb_drive
-from storage import getmount
-from storage import remount
+from storage import (
+    disable_usb_drive, 
+    getmount, 
+    remount
+)
+
 from supervisor import disable_autoreload
-from gc import collect
-print("----------------\nL",end="")
+
+print("{- * 16}\nL", end="")
```

Comment:
This doesn't work like that in CircuitPython, it has to be 
`print('-' * 16 + "\nL")`